### PR TITLE
fix(popover): arrow. pointAtCenter misalignment error

### LINKED ISSUE #50221: Popover with `pointAtCenter: true` does not align properly
### Reproduction link

[Edit on StackBlitz](https://stackblitz.com/edit/react-wthf3a?file=demo.tsx)

### Steps to reproduce

image

### What is expected?

The arrow is aligned with the center of the element.

### What is actually happening?

The arrow is not aligned with the center of the element.

| Environment | Info |
| --- | --- |
| antd | 5.19.4 |
| React | 18.3.0 |
| System | macOS |
| Browser | Chrome |

### PR DESCRIPTION
**Pull request by Wxh16144**
[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)

### 🤔 This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 🔗 Related issue link

fix: #50221

### 📝 Changelog

- Use a developer-oriented tone and narrative style.
- Describe the user's first-hand experience of the issue and its impact on developers, rather than your solution approach.
- Refer to: https://ant.design/changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |     fix(popover): `arrow. pointAtCenter` misalignment error      |
| 🇨🇳 Chinese |     修复 Popover 设置  `arrow.pointAtCenter` 属性时箭头未对齐错误     |

### CODE REVIEW

**Comment by Wxh16144** [comment]
查看 debug demo: https://preview-50260-ant-design.surge.sh/~demos/popover-demo-arrow-point-at-center

File: components/popover/demo/arrow-point-at-center.tsx [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+      <div className={styles.container} />
+    </Popover>
+  );
+}
+
+export default () => (
+  <Wrapper>
+    <Demo />
+  </Wrapper>
+);
```
Comment (afc163) [inline components/popover/demo/arrow-point-at-center.tsx:128]: 这个 demo 直接平铺 + open 吧，这样好截图。

**Comment by afc163** [comment]
用例挂了

**Comment by afc163** [comment]
4.x 也是挂的，干脆顺便一起修一下 https://github.com/ant-design/ant-design/issues/37817

**Comment by Wxh16144** [comment]
> 4.x 也是挂的，干脆顺便一起修一下 #37817

4.x 不是不接受 PR 维护了么

**Comment by afc163** [comment]
我们可以主动修。

**Comment by afc163** [comment]
ci 挂了，看看

File: components/popover/demo/arrow-point-at-center.tsx (hunk 1 above)
Comment (Wxh16144) [reply components/popover/demo/arrow-point-at-center.tsx]: > 这个 demo 直接平铺 + open 吧，这样好截图。

我决定就保持现状了， 视觉测试还是有很多需要补充的，这个 PR 先修复问题，附上一个 debug demo。

**Comment by afc163** [comment]
ci 挂了，看看

File: components/popover/demo/arrow-point-at-center.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+## zh-CN
+
+`arrow={{ pointAtCenter: true }}` 属性可以让箭头指向目标元素的中心。
```
Comment (afc163) [inline components/popover/demo/arrow-point-at-center.md:3]: xxx

**Review by afc163 — APPROVED** [review]

**Pull request merged**